researchsuite-sdk/build.gradle

```diff
@@ -0,0 +1,46 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 26
+    buildToolsVersion "26.0.2"
+
+
+    defaultConfig {
+        minSdkVersion 16
+        targetSdkVersion 26
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+
+        javaCompileOptions {
+            annotationProcessorOptions {
+                arguments = ["room.schemaLocation": "$projectDir/schemas".toString()]
+            }
+        }
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile 'com.android.support:appcompat-v7:26.1.0'
+
+    compile 'org.sagebionetworks.researchsuite:api:0.0.3'
```

Comment:
Where can I find documentation on this library?

Reply:
Here's a (outdated) diagram here for Android: https://github.com/Sage-Bionetworks/ResearchSuite-Android/pull/1

Shannon and I are working on documenting the high level design. Shannon would be the best resource on what the end result/goals look like. For now it'd be good to follow pull requests here as we flesh things out:
https://github.com/Sage-Bionetworks/ResearchSuite-Android/
https://github.com/Sage-Bionetworks/ResearchSuite-AppleOS/